### ISSUE TITLE
[REVIEW]: BrainGlobe Atlas API: a common interface for neuroanatomical atlases

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2668]
**Submitting author:** @adamltyson (<a href="http://orcid.org/0000-0003-3225-1130">Adam Luke Tyson</a>)
**Repository:** <a href="https://github.com/brainglobe/bg-atlasapi/" target ="_blank">https://github.com/brainglobe/bg-atlasapi/</a>
**Version:** v1.0.0
**Editor:** @oliviaguest
**Reviewer:** @typically, @vitay
**Archive:** <a href="https://doi.org/10.5281/zenodo.4065389" target="_blank">10.5281/zenodo.4065389</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/777458c96dffe8499c0d13886ecfe9bf/status.svg)](https://joss.theoj.org/papers/777458c96dffe8499c0d13886ecfe9bf)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/777458c96dffe8499c0d13886ecfe9bf"><img src="https://joss.theoj.org/papers/777458c96dffe8499c0d13886ecfe9bf/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/777458c96dffe8499c0d13886ecfe9bf/status.svg)](https://joss.theoj.org/papers/777458c96dffe8499c0d13886ecfe9bf)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/brainglobe/bg-atlasapi/tree/joss" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@typically & @vitay, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @oliviaguest know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/brainglobe/bg-atlasapi/tree/joss">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@adamltyson) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/brainglobe/bg-atlasapi/tree/joss">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@adamltyson) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [x] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @typically, @vitay it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2668 with the following error: 

 /app/vendor/ruby-2.4.4/lib/ruby/2.4.0/find.rb:43:in `block in find': No such file or directory - tmp/2668 (Errno::ENOENT)
	from /app/vendor/ruby-2.4.4/lib/ruby/2.4.0/find.rb:43:in `collect!'
	from /app/vendor/ruby-2.4.4/lib/ruby/2.4.0/find.rb:43:in `find'
	from /app/vendor/bundle/ruby/2.4.0/bundler/gems/whedon-132474d2711b/lib/whedon/processor.rb:61:in `find_paper_paths'
	from /app/vendor/bundle/ruby/2.4.0/bundler/gems/whedon-132474d2711b/bin/whedon:50:in `prepare'
	from /app/vendor/bundle/ruby/2.4.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /app/vendor/bundle/ruby/2.4.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /app/vendor/bundle/ruby/2.4.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /app/vendor/bundle/ruby/2.4.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /app/vendor/bundle/ruby/2.4.0/bundler/gems/whedon-132474d2711b/bin/whedon:119:in `<top (required)>'
	from /app/vendor/bundle/ruby/2.4.0/bin/whedon:23:in `load'
	from /app/vendor/bundle/ruby/2.4.0/bin/whedon:23:in `<main>'

**Comment by oliviaguest** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: :point_left:

**Comment by oliviaguest** [comment]
Hey 👋 @typically, @vitay: this is where the review will take place. Please make sure to read the instructions above.

For any and all things worthy of discussion or comment, use this issue right here — so drop comments or questions for me, the author, etc., here. For any very code-specific things please feel free to start an issue on the repo of the code itself (if appropriate!) and link back to it from here. For an example of how this process plays out feel free to skim previous reviews, such as: #2285 and #2348. ☺️

**Comment by vitay** [comment]
@adamltyson just a small comment on the paper: the Winnubst et al. 2019 reference could use a doi.

**Comment by oliviaguest** [comment]
@whedon check references

**Comment by oliviaguest** [comment]
@whedon check references

**Comment by oliviaguest** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: :point_left:

**Comment by oliviaguest** [comment]
@openjournals/dev why is the check references command not working? Is it related to the fact it's on a custom branch?

**Comment by danielskatz** [comment]
@whedon check references from branch joss

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch joss
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.3389/fninf.2015.00011 is OK
- 10.1038/s41467-019-13057-w is OK
- 10.1002/cne.24080 is OK
- 10.1002/cne.24080 is OK
- 10.1073/pnas.0604911103 is OK
- 10.7554/eLife.53350 is OK
- 10.1016/j.cell.2020.04.007 is OK
- 10.1016/j.neuron.2019.04.034 is OK
- 10.1101/2020.02.23.961748 is OK
- 10.5281/zenodo.3991718 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by FedeClaudi** [comment]
Hi @vitay  I've added the DOI:  https://github.com/brainglobe/bg-atlasapi/commit/dbe6a38ab3a8d692cfd16f74c88e086347fee1d2, thank you for pointing that out.

**Comment by vitay** [comment]
I guess I am already done with the review:
* `bg-atlas-api` is a software with a very clear rationale, providing a simple centralized access to various neuroatanomical atlases. It makes mostly sense within the brainglobe suite, but its API is generic enough so that other tools can build on it. 
* The software respects all standards and good practice: hosting on github, BSD license, unittests, CI on travis, online documentation, community guidelines, etc. I have not read the complete code, but the Python code is readable and respects conventions.
* Installation works out of the box with pip (tested with python 3.7 and 3.8), with only a handful of dependencies.
* The promised functionality is there. I have had troubles installing the other elements of the suite (e.g. brainrender-gui) but I could interact with the library directly, using the provided notebook as a basis.
*  The companion paper correctly describes the scope of the software.

In short, I recommend acceptance.

**Comment by oliviaguest** [comment]
Hi @typically can you give us an ETA for your review, please? ☺️

**Comment by typically** [comment]
> Hi @typically can you give us an ETA for your review, please? ☺️

Hi @oliviaguest, I should be able to get to this later next week, once students get sorted (it's orientation week here). It's on my radar!

**Comment by oliviaguest** [comment]
@whedon remind @typically in 16 days

**Comment by whedon** [comment]
Reminder set for @typically in 16 days

**Comment by oliviaguest** [comment]
@typically OK, I set-up a realistic reminder (for both of us to stay on track) given your plan then — thank you. 😊

**Comment by typically** [comment]
Okay, done reviewing. In summary, I love the tool, especially its simplicity and extensibility. I could definitely see how this could be integrated into any number of projects, including my own.

- Nice simple tool
- Excellent documentation
- Well organized Github page
- Easy to install (pip 20.1.1)
- Paper is readable and makes a good case for need

There are good basic examples of how to use the tool, but as minor suggestions:

1. It might be useful to demonstrate with a practical example how to deal with coordinates (Cartesion, image indices). There is a reference to another package, but it isn't super clear to me how I would go from a downloaded atlas to some standard coordinate system.
2. Hierarchical information appears to be provided (in the form of "structure_id_path"), and discussed in the paper. It would be useful to provide an example of how to use this information to obtain parcellations at different levels of hierarchy. It is not clear to me how this information can be represented, as the "annotation" image contains only a single integer per voxel.

I also recommend acceptance.

**Comment by adamltyson** [comment]
Hi @typically, thanks for your review!

For your first suggestion, I'm not 100% clear what you meant, do you mean something like:
1) How the voxel spacing in  the images corresponds to the "real world"? i.e. showing that you can query the different resolution atlases in microns, and get the same results? And also show that the generated meshes for each brain region are in micron units (not voxels).

or

2) How to get your own data into the standard coordinate system? e.g. using [brainreg](https://github.com/brainglobe/brainreg) to register sample image data to the atlas template image, and then analyse your data in the common coordinate space?

**Comment by typically** [comment]
Hi @adamltyson, I'm thinking of human templates, but more generally stereotaxic spaces. How would one map an image to Cartesian coordinates such as MNI-152, Talairach space, or rat stereotaxic coordinates, relative to bregma? You would need an origin and a basis vector, I assume. Just a suggestion though!

**Comment by oliviaguest** [comment]
@adamltyson I will leave you to think about any changes you might want to make before getting this officially accepted and published. So just drop a message here if you plan to do that, etc. 💯

**Comment by adamltyson** [comment]
Thanks for clarifying @typically. We don't have any method for converting between coordinate systems, but this would certainly be useful. Currently we only have one coordinate system for each species (e.g. each of the mouse atlases uses the same coordinates). If there's interest (and data available) for atlases with additional coordinate spaces, we could also provide methods to transform coordinate between coordinate spaces. 

So, if it's alright with you, suggestion 1 will go onto our todo list, but it will depend on which atlases we add. We will document suggestion 2, as these methods exist in the software, but are not fully documented yet.

@oliviaguest, we'll add these small changes to the tutorials, then I think we'll be ready to publish.

**Comment by typically** [comment]
Hi @adamltyson, sounds good to me! Thanks for making this tool available. :)

**Comment by typically** [comment]
(sorry, closed inadvertently)

**Comment by vigji** [comment]
@typically now both the [tutorial notebook](https://github.com/brainglobe/bg-atlasapi/blob/master/tutorials/Atlas%20API%20usage.ipynb) and the `Usage` section of the docs [here](https://docs.brainglobe.info) discuss more diffusely the hierarchy. As for the mapping from one atlas to the other, pure translations of the origin and changes in resolution and axes order can be done using [https://github.com/brainglobe/bg-space](url), but we currently don't provide general support from affine/nonaffine transformations between spaces.

**Comment by adamltyson** [comment]
Hi @oliviaguest, as long as @vitay and @typically are happy, I think we're good to go now!

**Comment by typically** [comment]
@vigji - Looks great! Re. spaces, was just curious but no need to add if it's not yet supported! Quite happy for it to be published.

**Comment by adamltyson** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: :point_left:

**Comment by adamltyson** [comment]
We've made some small edits to the paper (affiliations, acknowledgments, reference formatting), and we're now good to publish. @oliviaguest what comes next?

**Comment by oliviaguest** [comment]
@adamltyson perfect — firstly, you need to deposit the code, so create an archive (on Zenodo, figshare, or other) and post the archive DOI here.

**Comment by adamltyson** [comment]
Thanks @oliviaguest, here's the DOI: [`10.5281/zenodo.4065389`](https://doi.org/10.5281/zenodo.4065389)

**Comment by oliviaguest** [comment]
@whedon set 10.5281/zenodo.4065389 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.4065389" target="_blank">10.5281/zenodo.4065389</a> is the archive.

**Comment by oliviaguest** [comment]
Great! @adamltyson I think the title of the repo on zenodo has to be the same as the title of the JOSS paper — change that when you get a moment. 😊

**Comment by oliviaguest** [comment]
@whedon check references

**Comment by adamltyson** [comment]
> Great! @adamltyson I think the title of the repo on zenodo has to be the same as the title of the JOSS paper — change that when you get a moment. 

Done

**Comment by oliviaguest** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
PDF failed to compile for issue #2668 with the following error: 

 Can't find any papers to compile :-(

**Comment by oliviaguest** [comment]
@whedon generate pdf from branch joss

**Comment by whedon** [comment]
```
Attempting PDF compilation from custom branch joss. Reticulating splines etc...
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.02668/joss.02668/10.21105.joss.02668.pdf) :page_facing_up: :point_left:

**Comment by oliviaguest** [comment]
@whedon check references from branch joss

**Comment by whedon** [comment]
```
Attempting to check references... from custom branch joss
```

**Comment by oliviaguest** [comment]
@whedon set v1.0.0 as version

**Comment by whedon** [comment]
OK. v1.0.0 is the version.

**Comment by oliviaguest** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #2668 with the following error: 

 Can't find any papers to compile :-(

**Comment by oliviaguest** [comment]
@whedon accept from branch joss

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1779

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1779, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true from branch joss 
```

**Comment by arfon** [comment]
@whedon accept deposit=true from branch joss

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1313065522490155010) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1780
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02668](https://doi.org/10.21105/joss.02668)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by adamltyson** [comment]
Awesome, thanks @oliviaguest, @typically, @vitay! 

@arfon - We've been very (very) impressed with the JOSS process, thanks!

**Comment by FedeClaudi** [comment]
Yes, thank you very much to everyone involved. 
And I can confirm, the submission process as JOSS was very impressive!

**Comment by arfon** [comment]
@typically, @vitay - many thanks for your reviews here and to @oliviaguest for editing this submission ✨ 

@adamltyson @FedeClaudi - your paper is now accepted into JOSS :zap::rocket::boom:

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02668/status.svg)](https://doi.org/10.21105/joss.02668)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02668">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02668/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02668/status.svg
   :target: https://doi.org/10.21105/joss.02668
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02668/status.svg)](https://doi.org/10.21105/joss.02668)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss

**Comment by adamltyson** [comment]
Hi @arfon, I just noticed the link to the repo on the JOSS page links to the branch with the paper on (that won't be updated), rather than the link at the top of this issue ([https://github.com/brainglobe/bg-atlasapi/](https://github.com/brainglobe/bg-atlasapi/)). Is it possible for this to be changed?

**Comment by arfon** [comment]
Fixed!
